requirements.in

```diff
@@ -70,4 +70,4 @@ uwsgi==2.0.22
 uwsgi-readiness-check==0.2.0
 django-allow-cidr==0.6.0
 django-csp==3.7
-django-ansible-base @ git+https://github.com/ansible/django-ansible-base@2025.1.31#egg=django-ansible-base[jwt_consumer,resource-registry]
+django-ansible-base==2025.3.7
```

Comment:
what this is going to install, as per the missing dependencies usage pyjwt, requests ...
think we are missing our dependencies from this library e.g. 
[jwt-consumer,resource-registry]

Reply:
I don't see any differences in the final files, except `dynaconf` being pulled in now.

Reply:
https://github.com/ansible/django-ansible-base/blob/devel/docs/Installation.md